### DISCUSSION TITLE
Responsive ratio in next BS version?

### DISCUSSION THREAD

**Discussion by mikaelbianco** [twbs/bootstrap#36243] [Ideas] [answered]
Something like this:
[https://michalbialkowski.github.io/mdb/custom-ratio.html](https://michalbialkowski.github.io/mdb/custom-ratio.html)

**Comment by mikaelbianco** [comment]
![11DA799C-FC44-4BDB-8A88-7EB5D0285F99](https://user-images.githubusercontent.com/23509648/165873037-dc9f6dbe-9882-44e5-8819-9ea603c5d852.png)
![84BD7E72-1797-4FEE-8E8F-560B5DDA6E02](https://user-images.githubusercontent.com/23509648/165873040-b264a0b2-4012-4a16-9929-ac5eaaec2233.png)

**Comment by mdo** [comment, marked as answer]
Maybe next major version when we can move from a helper to a utility using some newer CSS. Safari just recently added support for `aspect-ratio`: https://caniuse.com/mdn-css_properties_aspect-ratio.

**Reply by mikaelbianco** [reply]
Nice! :-)

So far I solve it this way: 
https://codepen.io/micha-biakowski/pen/RwQbLxj
